assets/src/components/nav/leftNav.tsx

```diff
@@ -235,26 +241,40 @@ const ViewToggle = ({
   viewIsOpen,
   toggleView,
   collapsed,
+  disabled,
 }: {
   icon: JSX.Element
   name: string
   viewIsOpen: boolean
   toggleView: () => void
   collapsed: boolean
+  disabled?: boolean
 }): JSX.Element => {
-  return (
+  const buttonContent = (
     <button
       className={
         "m-left-nav__link m-left-nav__view" +
-        (viewIsOpen ? " m-left-nav__view--active" : "")
+        (viewIsOpen ? " m-left-nav__view--active" : "") +
+        (disabled ? " m-left-nav__view--disabled" : "")
       }
       onClick={toggleView}
       title={name}
+      aria-disabled={disabled}
     >
       {icon}
       {collapsed ? null : name}
     </button>
   )
+
+  if (disabled) {
+    return (
+      <Tippy content="Not available in Search Map" placement="right">
```

Comment:
thought (non-blocking): this seems to bake in an assumption about the reason for why the button is disabled. Seems okay for now while there is only one reason a button would be disabled, but maybe something that we'd want to think about parameterizing down the line.

Reply:
Yeah, that's a good point. Maybe instead of just taking a `disabled` prop of type boolean it could take an optional `disabledReason` prop that, if specified, would both disable the button and also add the string given as the contents of the tooltip.